<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT" verbose="0">
  <listeners>
    <listener class-name="loneCreates.resources.Listeners"/>
    <listener class-name="loneCreates.resources.TestRetryAnalyzerListener"/>
  </listeners>
  <test thread-count="5" parallel="tests" name="openItems(failed)" verbose="0">
    <classes>
      <class name="loneCreates.ItemsTest">
        <methods>
          <include name="Addnewitemtest"/>
          <include name="tearDown"/>
          <include name="driverInitialization"/>
          <include name="ItemExists"/>
        </methods>
      </class> <!-- loneCreates.ItemsTest -->
    </classes>
  </test> <!-- openItems(failed) -->
  <test thread-count="5" parallel="tests" name="LoginTest(failed)" verbose="0">
    <classes>
      <class name="loneCreates.ValidLoginTest">
        <methods>
          <include name="validLogin"/>
          <include name="tearDown"/>
          <include name="driverInitialization"/>
        </methods>
      </class> <!-- loneCreates.ValidLoginTest -->
      <class name="loneCreates.InvalidLoginTest">
        <methods>
          <include name="InvalidLogin"/>
          <include name="tearDown"/>
          <include name="driverInitialization"/>
        </methods>
      </class> <!-- loneCreates.InvalidLoginTest -->
    </classes>
  </test> <!-- LoginTest(failed) -->
</suite> <!-- Failed suite [Suite] -->
